guides/plugins/plugins/framework/caching/index.md

```diff
@@ -0,0 +1,250 @@
+
+```php
+<?php
+
+namespace MyProject;
+
+use Shopware\Core\Content\Product\Events\ProductIndexerEvent;
+use Shopware\Core\Content\Product\Events\ProductNoLongerAvailableEvent;
+use Shopware\Core\Framework\DependencyInjection\CompilerPass\RemoveEventListener;
+use Shopware\Core\Framework\Adapter\Cache\CacheInvalidationSubscriber;
+
+class TweakCacheInvalidation implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        $container
+            ->getDefinition(CacheInvalidationSubscriber::class)
+            ->clearTag('kernel.event_listener')
+    }
+
+}
+```
+
+However, suppose only certain parts of the cache invalidation are to be adjusted, finer adjustments to the class can be made using `Shopware\Core\Framework\DependencyInjection\CompilerPass\RemoveEventListener`, in which it is possible to define which event listeners of the service are to be removed.
+
+```php
+<?php
+
+namespace MyProject;
+
+use Shopware\Core\Content\Product\Events\ProductIndexerEvent;
+use Shopware\Core\Content\Product\Events\ProductNoLongerAvailableEvent;
+use Shopware\Core\Framework\DependencyInjection\CompilerPass\RemoveEventListener;
+use Shopware\Core\Framework\Adapter\Cache\CacheInvalidationSubscriber;
+
+class TweakCacheInvalidation implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        RemoveEventListener::remove(
+            $container,
+            CacheInvalidationSubscriber::class,
+            [
+                [ProductIndexerEvent::class, 'invalidateListings'],
+                [ProductNoLongerAvailableEvent::class, 'invalidateListings'],
+            ]
+        );
+    }
+} 
+```
+
+### Object Cache
+
+The internal caches are built upon the [Symfony Cache](https://symfony.com/doc/current/components/cache.html) component and are used internally to cache data that is expensive to compute or retrieve.
+As the object caches are handled internally it should not be needed to control them directly, therefore adding custom tags or manipulating the cache key is not supported for the various object caches.
+
+#### Cache invalidation
+
+However, you can still manually invalidate the object caches, via the same mechanism as you invalidate the HTTP-Cache, the `CacheInvalidator` service.
+
+You can use the `CacheInvalidator` service to invalidate the object caches by passing the tag you want to invalidate.
+
+```php
+public function invalidateSystemConfigCache(): void
+{
+    $this->cacheInvalidator->invalidate([
+        CachedSystemConfigLoader::CACHE_TAG
+    ]);
+}
+```
+
+### Delayed Invalidation
+
+By default, the cache invalidation happens delayed (for both http and object caches). This means that the invalidation is not instant, but rather all the tags that should be invalidated are invalidated in a regular interval.
```

Comment:
makes sense to mention here that you can force it in API request level

Reply:
Isn´t this covered by https://github.com/shopware/docs/pull/1743/commits/66d3fb746d647f7b6dedaa212194ef32be5c9c92#diff-5d93842ffdb748b64806380c0b37a0b231f9b67f9c3ff01e60416317406b354cR238 ?

Reply:
yes, i now linked from the intro directly to how force invalidate